File: go/http-server.go
package main

import (
	"fmt"
    "net/http"
)

func getFibonacci(w http.ResponseWriter, req *http.Request) {
	fmt.Fprintf(w, "result is %v", fibonacci(40))
}

func fibonacci(n uint) uint {
	if n < 2 {
		return n
	}
	return fibonacci(n-1) + fibonacci(n-2)
}

func main() {
    http.HandleFunc("/fibonacci", getFibonacci)
    
	fmt.Println("Go web server running in http://localhost:3003")
	http.ListenAndServe(":3003", nil)
}

File: nodejs/http-server.js
const http = require('http')
const PORT = 3001

const fibonacci = n => n < 1 ? 0 : n <= 2 ? 1 : fibonacci(n - 1) + fibonacci(n - 2)

http.createServer((req, res) => {
	const result = fibonacci(40)
	res.write(`result is ${result}`)
	res.end()
}).listen(PORT, () => {
	console.log(`NodeJs http server is running on http://localhost:${PORT}`)
})
